aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-3.md

```diff
@@ -31,13 +31,13 @@ The above change indicates that instead of returned a string, we instead want to
 
 We'll now add an appropriate View template to our project. To do this we'll position the text cursor within the Index action method, then right-click and select "Add View". This will bring up the Add View dialog:
 
-![](mvc-music-store-part-3/_static/image1.jpg) ![](mvc-music-store-part-3/_static/image1.png)
+![Small screenshot of the menu that shows the add view selection.](mvc-music-store-part-3/_static/image1.jpg) ![Screenshot of the add view dialog box, with menu options for selecting and adding your view.](mvc-music-store-part-3/_static/image1.png)
```

Comment:
Remove "small" screenshot - unnecessary information. Try to get this to be at least 100 characters. Also, put the second alt text and image on its own line.